conman/src/scheme.cpp

```diff
@@ -1759,6 +1802,39 @@ bool Scheme::disableBlocks(
   return success;
 }
 
+bool Scheme::disableBlocksTopo(
```

Comment:
?Ok I'll do that now!

________________________________
From: Jonathan Bohren <notifications@github.com>
Sent: Wednesday, February 11, 2015 11:33 AM
To: jbohren/conman
Cc: Rose Wall
Subject: Re: [conman] enableBlocksTopo and disableBlocksTopo functions added (#21)


In conman/src/scheme.cpp<https://github.com/jbohren/conman/pull/21#discussion_r24509108>:

> @@ -1759,6 +1802,39 @@ bool Scheme::disableBlocks(
>    return success;
>  }
>
> +bool Scheme::disableBlocksTopo(


In this PR, can you just replace disableBlocks with the contents of disableBlocksTopo?

-
Reply to this email directly or view it on GitHub<https://github.com/jbohren/conman/pull/21/files#r24509108>.